pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,127 @@
+
+A block is a container image that can be added to a multicontainer fleet in order to provide specific functionality. Blocks are intended to reduce the friction of app development by making it easier to re-use functionality, much like software packages. 
+
+Similar to how software developers don't want to keep rewriting common functions, IOT developers don't want to keep creating containers for common tasks, such as handling Bluetooth connections or interacting with attached displays. Blocks provide solutions to common IoT application requirements, which developers can re-use to enable their fleet.
+
+For example, getting a browser to run well in a container is a tedious task. Hence, why not use the [browser block][browser] to things up and running faster.
+
+<img src="/img/blocks/diagram-1.png" width="100%">
+
+## Getting Started with blocks
+
+Blocks being a container image can be added to a multicontainer fleet. To use a block in your fleet, add the block's image as a new service in the `services` section of your app's `docker-compose.yml` file. This is similar to how we would build [multicontainer fleets][multicontainer].
+
+Here's an example of using the [browser block][browser] in a sample multicontainer fleet already running 2 services.
+
+```
+version: '2'
+
+services:
+  # Browser block service added to the multicontainer fleet
+  browser:
+    image: balenablocks/browser
+    privileged: true
+    network_mode: host
+
+  # Existing services in the multicontainer app
+  balena-node-hello-world:
+    build: ./balena-node-hello-world
+    restart: always
+    depends_on:
+      - browser
+    ports:
+      - "80"
+  importantBananaDb:
+    image: importantBanana/db:latest
+    ports:
+      - "8080"
+    environment:
+      DB_ROOT_PASSWORD: 'potassium'
+      RUN_MODE: 'super-secure'
+```
+
+Each block will have be having their own set of [docker-compose fields][docker-compose] that's required for configuration and usage. Do take care of syntax and conflicts in the docker-compose.yml file when composing and deploying your multicontainer configuration.
+
+More resources:
+
+1. Video tutorial on [IoT Happy Hour](https://youtu.be/Mllay6Z2-qQ?t=1440) about using  balenaBlocks and building an [environmental sensor app](https://github.com/balenalabs-incubator/ruuvitag) using sensor, connector, and dashboard block.
+2. Check out our guide on [how to compose multicontainer fleets](https://www.balena.io/blog/two-projects-one-device-turn-your-raspberry-pi-into-a-multitool/) for more details.
+3. Blog on [Introduction to blocks](https://www.balena.io/blog/introducing-balenablocks-jumpstart-your-iot-app-development/).
+
+## Creating your own block
+
+Since blocks are container images, the process to create blocks is very similar to creating a container image. To learn more about the what can be a block, you can read this [blog post](https://www.balena.io/blog/how-to-make-your-own-balenablocks-simple-drop-in-edge-app-functionality/). 
```

Comment:
"since blocks are container images, the process to create them is very similar to creating any other container. To learn more about what makes a good candidate for a block...."